ChangeLog.md

```diff
@@ -1,4 +1,14 @@
 # ansible-powerstore Change Log
+## Version 1.3.0 - released on 28/09/2021
+- Added dual licensing.
+- Added CRUD operations for local user.
+- Network module is added to get the details and modify attributes of network.
+- Cluster module is added to get the details and modify attributes of cluster.
+- Job module is added to get the details of given job ID.
+- Role module is added to get the details of given role name or role ID.
+- Gather facts module is enhanced to list users, roles, networks and appliances.
+- Added dual licensing.
```

Comment:
Remove it - "dual licensing" mentioned twice.

Reply:
Addressed. Thanks.